src/Pages/IndexPage/IndexPage.js

```diff
@@ -0,0 +1,68 @@
+import React, { useContext, useEffect, useState } from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import Grid from "@material-ui/core/Grid";
+import AddNoteBoard from "./Comps/AddNoteBoard";
+import IndexPageContext from "../../Context/IndexPageContex";
+import useGetUserNotes from "../../Hooks/useGetUserNotes";
+import moment from "moment";
+import Note from "./Comps/Note";
+import AccountCircleIcon from "@material-ui/icons/AccountCircle";
+const useStyles = makeStyles((theme) => ({
+  root: { padding: 20, width: "100%" },
+}));
+
+const IndexPage = () => {
+  const classes = useStyles();
+  const { indexPageState } = useContext(IndexPageContext);
+  const [getUserNotes] = useGetUserNotes();
+
+  const [notes, setNotes] = useState([]);
+  useEffect(() => {
+    getUserNotes();
+  }, []);
+  useEffect(() => {
+    setNotes(indexPageState.notes);
+  }, [indexPageState.notes]);
+
+  return (
+    <Grid
+      container
+      direction="column"
+      justify="center"
+      alignItems="center"
+      className={classes.root}
+    >
+      <Grid item>
+        <Grid container direction="column" justify="center" alignItems="center">
+          <Grid item>
+            <AddNoteBoard />
+          </Grid>
+          <Grid item>
+            <Grid
+              container
+              direction="column-reverse"
+              justify="center"
+              alignItems="center"
+            >
+              {notes.map((item) => {
+                return (
+                  <Note
```

Comment:
Maybe just pass in note={item} and do all of this inside?
// Note.js
const { title, content, color, etc } = note;